packages/bitget/src/index.ts

```diff
@@ -1,39 +1,39 @@
 import type { WalletInit, EIP1193Provider } from '@web3-onboard/common'
 
-let bitgetDependencies: any = null
-let bitgetAdapter: any = null
+let bitgetWalletDependencies: any = null
+let bitgetWalletAdapter: any = null
 
-const loadBitgetDependencies = async (getAdapter?: boolean) => {
-  if (!bitgetDependencies) {
-    bitgetDependencies = await import('@bitget-wallet/web3-sdk')
+const loadBitgetWalletDependencies = async (getAdapter?: boolean) => {
+  if (!bitgetWalletDependencies) {
+    bitgetWalletDependencies = await import('@bitget-wallet/web3-sdk')
   }
-  if (getAdapter && !bitgetAdapter) {
-    bitgetAdapter = new bitgetDependencies.LegacyEip1193Adapter()
-    return bitgetAdapter
+  if (getAdapter && !bitgetWalletAdapter) {
+    bitgetWalletAdapter = new bitgetWalletDependencies.LegacyEip1193Adapter()
+    return bitgetWalletAdapter
   }
   if (getAdapter) {
-    return bitgetAdapter
+    return bitgetWalletAdapter
   }
-  return bitgetDependencies
+  return bitgetWalletDependencies
 }
 
-function bitget(): WalletInit {
+function bitgetWallet(): WalletInit {
   if (typeof window === 'undefined') return () => null
 
   return () => {
     return {
-      label: 'Bitget',
+      label: 'Bitget Wallet',
```

Comment:
This line will update the client naming of the wallet.
You can probably revert the changes to add `Wallet` to variable names as it will be implied

Reply:
I see. Thanks for reply